py-polars/polars/dataframe/frame.py

```diff
@@ -3219,6 +3220,148 @@ def write_database(
         else:
             raise ValueError(f"'engine' {engine} is not supported.")
 
+    def write_delta(
+        self,
+        table_or_uri: str | Path | deltalake.DeltaTable,
+        *,
+        mode: Literal["error", "append", "overwrite", "ignore"] = "error",
+        overwrite_schema: bool = False,
+        storage_options: dict[str, str] | None = None,
+        delta_write_options: dict[str, Any] | None = None,
+    ) -> None:
+        """
+        Write DataFrame as delta table.
+
+        Parameters
+        ----------
+        table_or_uri
+            URI of a table or a DeltaTable object.
+        mode
+            How to handle existing data. Default is to error if table already exists.
+
+            * If 'append', will add new data.
+            * If 'overwrite', will replace table with new data.
+            * If 'ignore', will not write anything if table already exists.
+        overwrite_schema
```

Comment:
Is there a reason you included this argument directly? It can also be specified as part of `delta_write_options`. Do you think it will be used frequently enough to warrant being a separate parameter?

Reply:
yes, this is one of those args used quite a lot, like `mode`. 

since delta-rs doesn't support schema evolution yet, this option has to be used along with `mode="overwrite"`